### ISSUE TITLE
Journal as evidence of your work

### ISSUE THREAD

**Issue by NMITTutor** [bob-donnelly/journals-sdv-602#1]
Kia Ora Bob
Your week two journal is missing. 
Also could you please submit your "homework" exercises to be done as presented in the "lecture notes"- into your journal or as links from your journal. 

BTW - I wont give permissions to third party S/W to act on my behalf on GitHUB.

**Comment by bob-donnelly** [comment]
> 
> 
> Kia Ora Bob
> Your week two journal is missing.
> Also could you please submit your "homework" exercises to be done as presented in the "lecture notes"- into your journal or as links from your journal.
> 
> BTW - I wont give permissions to third party S/W to act on my behalf on GitHUB.

Hi Todd,

I have been attempting the exercises for the Friday class and am struggling with them. Instead of pushing an incomplete journal with no exercises attempted I am trying to solve some of them before uploading the journal with the first half of the week that I have completed. 

I read and re-read the instructions many times since Friday before I made progress but since I was making progress I ended up not reaching out for help.

Feel free not to use the third party S/W if you do not want to. It is an option I put there as an exercise for myself.
